packages/engine/Source/Scene/ScreenSpaceCameraController.js

```diff
@@ -2189,10 +2191,10 @@ function zoom3D(controller, startPosition, movement) {
     zoom3DCartographic
   ).height;
 
-  const inertiaMovementApproachingGround = Math.abs(height) < 50;
+  const approachingCollision = Math.abs(preIntersectionDistance) < 4000;
```

Comment:
Since this is an arbitrary value it's probably best to expose this value as a property, like what is done for `controller.minimumPickingTerrainHeight`. I would call it something like `minimumPickingTerrainHeightWithInertia`.